lib/linters/property_ordering.js

```diff
@@ -15,17 +15,18 @@ module.exports = {
             throw new Error('Invalid setting value for propertyOrdering: ' + config.style);
         }
 
-        node.walkDecls(function (declaration) {
+        node.each(function (child) {
```

Comment:
so why wasn't walkDecls working correctly here? was it because it was cascading into the declarations of nested stuffs?